### ISSUE TITLE
Display Tags on Dashboard

### ISSUE THREAD

**Issue by Rabenherz112** [opencve/opencve#484]
**Is your feature request related to a problem?**
In the previous version, we used tags to assign vulnerabilities to specific individuals and indicate the current status of each vulnerability. For example, a CVE might have tags like 'colleague1', 'new' or 'colleague1', 'In research'. However, in Version 2, tags are no longer displayed on the dashboard and are only visible when viewing the details of a CVE. This means we need to click on each CVE just to display the status.

**Do you have a solution in mind, or a suggestion to improve OpenCVE?**
Please make tags visible again on the dashboard.

**Additional comment**
N/A

**Comment by ncrocfer** [comment]
@Rabenherz112 I'm not sure to understand the issue, the tags are still available in the dashboard view (under the `Tags` panel), so you can easily click on it to filter the CVEs.

Can you share a screenshot to better understand?

**Comment by Rabenherz112** [comment]
Sure!
Here are some examples of what I mean. In the old version, when adding a tag, you could see the tags (such as assignment and state) directly, without needing to click on the CVE or search for a specific tag. This made it easy to identify, for example, new CVEs that were still unassigned. However, in the new version, even though the CVE has a tag, it isn’t displayed. This means you can no longer skim through the dashboard to quickly check if everything has been assigned (at least for our use-case).
![2024-12-18-13-52_1840](https://github.com/user-attachments/assets/f61c07e1-25bc-4a53-871d-ca911037b645)
![2024-12-18-13-53_1841](https://github.com/user-attachments/assets/cab425d5-19d5-4d40-8f81-12016cb932d0)

**Comment by Rabenherz112** [comment]
> You’d like the tags associated with each CVE to be displayed in the Activity Feed (specifically within the CVE changes section).

Yes, that is what I meant, sorry if I expressed myself somewhat unclearly there :D

> Could you share more details about the Colleague 1 and Colleague 2 tags you’re using?

Regarding the colleague's tags, we have one shared account and each team member who uses the system gets their own tag with their name. When a new CVE is published, either they assign it to themselves or it gets assigned to them (which means we add their name tag to the CVE). This allows us to track on the fly how many CVEs each member is currently working on, it clearly shows that the CVE are assigned to somebody, so no two people are working on it (or worse nobody) and it helps us identify who to approach if there are any questions or issues with a specific CVE, as we know who worked on it. So basically it is just there to ensure that there is no overlap or confusion about who is responsible for what.

**Comment by ncrocfer** [comment]
Thank you for sharing this information with us! It’s very clear, and it gives us valuable insights into how we might improve the memberships feature in OpenCVE.

In particular, the way you manage the dispatching of CVEs (assigning who is responsible for what) is inspiring. We’ll definitely consider this as we explore ways to enhance this aspect of the platform 👍
